File: remote/src/main/java/com/kelsos/mbrc/constants/ProtocolEventType.java
package com.kelsos.mbrc.constants;


public final class ProtocolEventType {

    private ProtocolEventType() { }

    public static final String INITIATE_PROTOCOL_REQUEST = "InitiateProtocolRequest";
    public static final String REDUCE_VOLUME = "ReduceVolume";
    public static final String HANDSHAKE_COMPLETE = "HandshakeComplete";
    public static final String INFORM_CLIENT_NOT_ALLOWED = "InformClientNotAllowed";
    public static final String INFORM_CLIENT_PLUGIN_OUT_OF_DATE = "InformClientPluginOutOfDate";
    public static final String NO_SETTINGS_AVAILABLE = "DisplayDialog";
    public static final String USER_ACTION = "UserAction";
    public static final String PLUGIN_VERSION_CHECK = "PluginVersionCheck";
}

File: app/src/main/kotlin/com/kelsos/mbrc/constants/ProtocolEventType.kt
package com.kelsos.mbrc.constants

object ProtocolEventType {
  const val InitiateProtocolRequest = "InitiateProtocolRequest"
  const val ReduceVolume = "ReduceVolume"
  const val HandshakeComplete = "HandshakeComplete"
  const val InformClientNotAllowed = "InformClientNotAllowed"
  const val PluginUpdateAvailable = "PluginUpdateAvailable"
  const val PluginUpdateRequired = "PluginUpdateRequired"
  const val UserAction = "UserAction"
  const val PluginVersionCheck = "PluginVersionCheck"
}
